### PR TITLE
:arrow_up: Upgrades dependency to support OSX Sierra

### PR DESCRIPTION
**Pull request by jmelberg-okta**
Fixes issue with PhantomJS causing `jasmine` tests to hang.

See: [PhantomJS not working on OSX Sierra](https://stackoverflow.com/questions/39739393/phantomjs-not-working-on-os-x-sierra)

Bumps CDN version to `1.13.0`

Verified:
  - Albatross
  - Sierra
  - El Capitan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,16 +67,16 @@ To use the CDN, include links to the JS and CSS files in your HTML:
 ```html
 <!-- Latest CDN production Javascript and CSS: 1.11.0 -->
```
Comment (samerfanek-okta) [inline README.md]: Nit: This comment needs to be updated as well.

File: package-lock.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@okta/okta-signin-widget",
-  "version": "1.13.0",
+  "version": "2.1.0",
```
Comment (samerfanek-okta) [inline package-lock.json]: Why are we bumping up this version?

File: package-lock.json [hunk 3]
```diff
@@ -725,20 +725,13 @@
       "dev": true
     },
     "clipboard": {
-      "version": "1.7.1",
-      "resolved": "https://registry.npmjs.org/clipboard/-/clipboard-1.7.1.tgz",
-      "integrity": "sha1-Ng1taUbpmnof7zleQrqStem1oWs=",
+      "version": "1.6.1",
+      "resolved": "https://registry.npmjs.org/clipboard/-/clipboard-1.6.1.tgz",
+      "integrity": "sha1-ZcW2VIEkZrD6q4Lca6Dx0vjkvlM=",
```
Comment (samerfanek-okta) [inline package-lock.json]: Is there a reason for this package downgrade?

Comment (samerfanek-okta) [inline package-lock.json]: Similar comment to ``tiny-emitter`` and ``jasmine-core`` below.

File: package-lock.json [hunk 4]
```diff
@@ -5412,6 +5317,88 @@
         }
       }
     },
+    "phantomjs-prebuilt": {
+      "version": "2.1.14",
```
Comment (samerfanek-okta) [inline package-lock.json]: Just checking, did you verify that this version works on earlier versions than OSX Sierra?

File: package-lock.json (hunk 2 above)
Comment (jmelberg-okta) [inline package-lock.json]: This was regenerated when we updated the dependencies. Current version is `2.1.0`

File: package-lock.json (hunk 3 above)
Comment (jmelberg-okta) [inline package-lock.json]: This is a strange change. I'll make sure to revert these back.

Comment (jmelberg-okta) [inline package-lock.json]: RE: `clipboard` -> Bumped version to `1.7.1`. This trickled down and updated `tiny-emitter`.

RE: `jasmine-core`: Oddly, `grunt-contrib-jasmine` depends on a previous version of `jasmine-core`.

Verified that the tests pass 👍

File: package-lock.json (hunk 4 above)
Comment (jmelberg-okta) [inline package-lock.json]: It appears that our `.travis.yaml` file specifies the OS to be `linux`, so I am not sure if we check different versions of OSX. Curious if we should update the script to include different OS versions?

It appears to be pretty straightforward:
https://docs.travis-ci.com/user/reference/osx/#OS-X-Version

**Pull request closed without merging**